bundles/com.espressif.idf.terminal.connector/src/com/espressif/idf/terminal/connector/launcher/IDFConsoleLauncherDelegate.java

```diff
@@ -337,6 +338,13 @@ public ITerminalConnector createTerminalConnector(Map<String, Object> properties
 		if (envMap.containsKey("PATH") && envMap.containsKey("Path")) { //$NON-NLS-1$ //$NON-NLS-2$
 			envMap.remove("Path"); //$NON-NLS-1$
 		}
+		// Adding esp-idf/tools in the path if it's missing (IEP-1668)
+		String idfToolsPath = Paths.get(IDFUtil.getIDFPath(), "tools").toAbsolutePath().toString(); //$NON-NLS-1$
```

Comment:
we have a constant in IDFConstants TOOLS_FOLDER.
It is not ideal because ideal will be to have a utility function in IDFUtil to return this path